platforms/ios/src/TangramMap/TGMarker.mm

```diff
@@ -57,13 +57,24 @@ - (instancetype)initWithMapView:(TGMapViewController *)mapView
     return self;
 }
 
-- (void)setStyling:(NSString *)styling
+- (void)setStylingString:(NSString *)styling
```

Comment:
Hmm I see what you mean. The reason for these names on iOS is that these are setters for a "property" (an obj-c thing), so an application wouldn't actually call "setStylingString", it would just call `marker.stylingString = aStylingString`. Do you think it would be better as `marker.stylingFromString = aStylingString`?